### ISSUE TITLE
[BUG] Fractor change '>' sign with '&gt;'

### ISSUE THREAD

**Issue by Oktopuce** [andreaswolf/fractor#259]
### Minimal PHP Code Causing Issue

In the code below, you have a ViewHelper `{f:uri.page(pageUid: 1, absolute: 'true') -> f:format.htmlentities()}` wich will be transformed in `{f:uri.page(pageUid: 1, absolute: 'true') -'&gt; f:format.htmlentities()}` I guess because it's between `<link>` tag.

```
<?xml version="1.0" encoding="utf-8"?>
<html xmlns:content="http://purl.org/rss/1.0/modules/content/"
      xmlns:atom="http://www.w3.org/2005/Atom"
      xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers"
      data-namespace-typo3-fluid="true">
		<channel>
			<link>{f:uri.page(pageUid: 1, absolute: 'true') -> f:format.htmlentities()}</link>
		</channel>
	</rss>
</html>
```

### Applied rules

No fractor specific rule required.

### Expected Behaviour

The ViewHelper `{f:uri.page(pageUid: 1, absolute: 'true') -> f:format.htmlentities()}` musn't be transformed like this  `{f:uri.page(pageUid: 1, absolute: 'true') -'&gt; f:format.htmlentities()}`, it should remain like it was before appliying fractor.

### Package Version

0.4.2

### PHP Version

8.2.0

**Comment by andreaswolf** [comment]
True, but AFAIK what we do here in TYPO3 is not really valid XML/HTML, thus this might be hard to change. First step would be to identify which component exactly messes stuff up here – my guess is that this already happens when the XML is read, so it's probably just a missing libxml parameter. Should be easy to spot with a simple test.

**Comment by d-s-codappix** [comment]
This also affects FlexForms, e.g. we update from 12 to 13 right now and get this:

```
-                 <numIndex index="1">Person->showSelected</numIndex>
+                <numIndex index="1">Person-&gt;showSelected</numIndex>
```

This was often the case with switchable controller actions, which we need to migrate in this case.
But FlexForms are also not always valid XML … people might have used `>` also in labels or other places within FlexForms.

This is just for your info. I'm fine with keeping this broken in case of FlexForms, and force people to manually change this back and think about the situation.

**Comment by simonschaufi** [comment]
I got budget for this quarter and will look into this if nobody else finds the spot where this happens

**Comment by simonschaufi** [comment]
I did some research and found out that it is the fault of the `\DOMDocument` class right here:

```php
$xml = $document->saveXML();
```
